frontend/src/app/libs/common-components/event-poover/event-poover.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
let's fire some @Output with updated Partial<Event> when user creates new event

Reply:
also create public methods `.open()` and `.close` to open or close this popover.
best use case for this component is something like
`<create-draft-event-popover #draftEventPopover (creating)="createDraftEvent($event)"></create-draft-event-popover>
<button (click)="draftEventPopover.open()">Some Button</button>`

Reply:
Please rename component to `create-draft-event-popover`